.github/workflows/trigger_staging_deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Trigger release to staging
```

Comment:
Why do we need this action exactly?

So we don't have to create PRs manually and instead fire them off from the actions tab?

Reply:
Yes, because the format of the commit message is important (it's where the version number comes from). It also allows us to specify any ref to deploy, which is slightly easier than doing it manually. 

Another advantage is it is extensible: I can imagine us automatically running smoke tests as a result of this or similar

Reply:
That just leaves one question. Do we want to prevent any other ways of getting code into master? Is that even possible?

Reply:
Everything has to go through PR, and any commits which don't have the right message won't trigger a deploy. I don't know that there's a way to disable manual PRs, especially since it will be using your personal token